lib/rspec/mocks/any_instance/stub_chain.rb

```diff
@@ -38,7 +37,7 @@ def invocation_order
         end
 
         def verify_invocation_order(rspec_method_name, *_args, &_block)
-          return if invocation_order[rspec_method_name].include?(last_message)
+          return if invocation_order.fetch(rspec_method_name, [nil]).include?(last_message)
```

Comment:
Can you add a spec showing what happens as a result of this change, where it fails in an expected way instead of the confusing "Undefined method `include?`" error?